docs/cli/local/walrus_local_install.md

```diff
@@ -0,0 +1,19 @@
+# walrus local install
+
+Install local Walrus
```

Comment:
Do we need explain what install do?

Reply:
It's like a dictionary for indexing commands. Include a simple explanation for the command. Important details about local mode could be explained by this issue. https://github.com/seal-io/docs/issues/149